CalibCalorimetry/CaloTPG/src/CaloTPGTranscoderULUT.cc

```diff
@@ -204,12 +204,23 @@ CaloTPGTranscoderULUT::getOutputLUTSize(const HcalTrigTowerDetId& id) const
       case HcalTopologyMode::TriggerMode_2016:
          return QIE8_OUTPUT_LUT_SIZE;
       case HcalTopologyMode::TriggerMode_2017:
+         if (id.ietaAbs() <= theTopology->lastHERing())
+            return QIE8_OUTPUT_LUT_SIZE;
+         else
+            return QIE10_OUTPUT_LUT_SIZE;
+      case HcalTopologyMode::TriggerMode_2017plan1:
```

Comment:
hello @matz-e
the case 2017plan1 
is the baseline for CMS at this stage: are you planning on implementing it any time soon?
At this moment, 

case HcalTopologyMode::TriggerMode_2017plan1:

would proceed to execute the:

case HcalTopologyMode::TriggerMode_2018:

block, which is a worse approximation of TriggerMode_2017plan1 than TriggerMode_2017 is.